libc/src/stdlib/CMakeLists.txt

```diff
@@ -268,18 +268,27 @@ if(LLVM_LIBC_INCLUDE_SCUDO)
   set(SCUDO_DEPS "")
 
   include(${LIBC_SOURCE_DIR}/../compiler-rt/cmake/Modules/AllSupportedArchDefs.cmake)
-  if(NOT (LIBC_TARGET_ARCHITECTURE IN_LIST ALL_SCUDO_STANDALONE_SUPPORTED_ARCH))
-    message(FATAL_ERROR "Architecture ${LIBC_TARGET_ARCHITECTURE} is not supported by SCUDO. 
+  
+  # scudo distinguishes riscv32 and riscv64, so we need to translate the architecture
+  set(LIBC_TARGET_ARCHITECTURE_FOR_SCUDO ${LIBC_TARGET_ARCHITECTURE})
```

Comment:
Based on this, would it make sense to split Risc-v 64/32 in our build system as well?

Reply:
According to https://github.com/llvm/llvm-project/blob/40e2bb5330840b56d452244f96e491b6530ce4bf/libc/cmake/modules/LLVMLibCArchitectures.cmake#L144

`LIBC_TARGET_ARCHITECTURE` actively reset `riscv64` to `riscv` soon upon definition, with a purpose of simplying the build system.

Reply:
alright, in that case this seems fine.